app/javascript/components/demo-indicator/component.spec.js

```diff
@@ -0,0 +1,27 @@
+import { mountedComponent, screen } from "test-utils";
```

Comment:
Replace old file instead of replacing new file, run lint

Reply:
this is in old file